src/Microsoft.DotNet.Interactive.VisualStudio/Microsoft.DotNet.Interactive.VisualStudio.csproj

```diff
@@ -42,8 +42,8 @@
 
   <ItemGroup>
     <EmbeddedResource Include="..\Microsoft.DotNet.Interactive\Resources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+      <GenerateSource>"true"</GenerateSource>
+      <LogicalName>Microsoft.DotNet.Interactive.Resources.resources</LogicalName>
```

Comment:
I think these could be converted to attributes i.e.

<EmbeddedResource Include="..\Microsoft.DotNet.Interactive\Resources.resx" GenerateSource="true" LogicalName="Microsoft.DotNet.Interactive.Resources.resources" />

Reply:
Thank you for the suggestion.

I will apply the proposed change in case more fixes are needed.